build/Targets/Packages.props

```diff
@@ -223,6 +223,7 @@
     <VsWebsiteInteropVersion>8.0.0.0-alpha</VsWebsiteInteropVersion>
     <vswhereVersion>1.0.71</vswhereVersion>
     <xunitVersion>2.2.0-beta4-build3444</xunitVersion>
+    <xunitanalyzersVersion>0.5.0</xunitanalyzersVersion>
```

Comment:
The problem I was having was with the new runners, specifically the "dotnet xunit" runner. The library itself I hit no issues with, as far as I can tell (aside from the new warnings produced by the analyzer, of course).